runtime/common/src/impls.rs

```diff
@@ -163,3 +165,56 @@ impl From<u128> for InstanceId {
 		Self(v)
 	}
 }
+
+impl Default for PermissionRoles {
+	fn default() -> Self {
+		Self {
+			admin: AdminPoolRoles::empty(),
+			tranches: Vec::new(),
```

Comment:
If we wanted to be clever, we could probably also store tranches as a u32 bitfield, since we'll at some point want to set an upper limit on tranches allowed

Reply:
We can definitely do this. My reasoning here was, that due to the u8 we will have at most 256 distinct tranches per pool. And iterating over this range seemed okish. But starting with 32 distinct tranches is fine for me.